README.md

```diff
@@ -56,39 +64,123 @@ Morpheus Feedback API
 - OpenShift (for production deployment)
 
 ---
+
 ## Installation and Setup
 
 1. Clone the Repository:
-    ```
-    git clone https://github.com/your-repo/morpheus-feedback-api.git
-    cd morpheus-feedback-api
-    ```
+   ```
+   git clone https://github.com/your-repo/morpheus-feedback-api.git
+   cd morpheus-feedback-api
+   ```
 2. Create a Virtual Environment:
-    ```
-    python -m venv venv
-    source venv/bin/activate
-    ```
+
+   ```
+   python -m venv venv
+   source venv/bin/activate
+
+   ```
 3. Install Dependencies:
-    ```
-    pip install -r requirements.txt
-    ```
-4. Set Up Configuration:
-
-    Update config.py file with the following variables:
-    ```
-    ARGILLA_API_URL=<your-argilla-api-url>
-    ARGILLA_API_KEY=<your-argilla-api-key>
-    ARGILLA_DATASET  = <your-argilla-dataset>
-    ARGILLA_WORKSPACE = <your-argilla-workspace>
-    ```
+
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+**Note:** To change the configuration, you must set the following environment variables before running the application.
+
+   ARGILLA_API_URL
+   ARGILLA_API_KEY
+   ARGILLA_DATASET
+   ARGILLA_WORKSPACE
+
+   Here is an example of how to set these variables in a terminal:
+   ```bash
+   export ARGILLA_API_URL="<your-argilla-api-url>"
+   export ARGILLA_API_KEY="<your-argilla-api-key>"
+   export ARGILLA_DATASET="<your-argilla-dataset>"
+   export ARGILLA_WORKSPACE="<your-argilla-workspace>"
+   ```
+
 ---
+
 ## Running Locally
 
-1. Start the Flask Service:
-    ```
-    python run.py
-    ```
-2. Access the API:
 
-    Visit ```http://localhost:5001``` to access the API.
+1. Set Up Argilla Container:
+
+   Create a Docker network for Argilla:
+
+   ```bash
+   docker network create argilla-net
+   ```
+
+   Start Elasticsearch container:
+
+   ```bash
+   docker run -d --name elasticsearch-for-argilla --network argilla-net \
+     -p 9200:9200 -p 9300:9300 \
+     -e "ES_JAVA_OPTS=-Xms512m -Xmx512m" \
+     -e "discovery.type=single-node" \
+     -e "xpack.security.enabled=false" \
+     docker.elastic.co/elasticsearch/elasticsearch:8.5.3
+   ```
+
+   Start Argilla Quickstart container:
+
+   ```bash
+   docker run -d --network argilla-net \
+     -e "ARGILLA_ELASTICSEARCH=http://localhost:9200" \
+     --name quickstart -p 6900:6900 \
+     argilla/argilla-quickstart:latest
+   ```
+
+   **Note:** Make sure you have both `quickstart` and `elasticsearch-for-argilla` containers running.
+   See https://docs.argilla.io/v2.0/getting_started/quickstart/ for more details.
+
+2. Access Argilla UI:
```

Comment:
please refer users to this link https://docs.v1.argilla.io/en/v2.8.0/getting_started/installation/deployments/docker-quickstart.html
for information on how to login